tests/h/streamer/filter_test.py

```diff
@@ -203,6 +233,14 @@ def get_randomized_filter(self):  # pragma: no cover
                         "https://example.org" + str(random()),
                     ],
                 },
+                {
+                    "field": "/group",
+                    "operator": "equals",
+                    "value": [
+                        "3jgSANNlEeebpLMf36MACw" + str(random()),
+                        "3jgSANNlEeebpLMf36MACw" + str(random()),
```

Comment:
I suspect it doesn't matter for the purpose of this test, but it looks like the values beginning "3jg..." are meant to be dummy annotation IDs. A realistic-looking groupid (aka. a "pubid") is much shorter.

Reply:
True, I will address that on a likely following up PR.